src/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -498,8 +475,10 @@ async Task<MsalIdTokenCacheItem> ITokenCacheInternal.GetIdTokenCacheItemAsync(Ms
         }
 
         /// <remarks>
-        /// Get accounts should not make a network call, if possible.
+        /// Get accounts should not make a network call, if possible. This can be achieved if 
+        /// all the environments in the token cache are known to MSAL.
```

Comment:
maybe add something about how they would be known already to MSAL...this seems a little magical in terms of the comment